apps/backoffice/src/components/services/subscriptions-migration/migration-manager.tsx

```diff
@@ -0,0 +1,196 @@
+import { ButtonWithLoader } from "@/components/buttons";
```

Comment:
can you add a top level comment about what this module does?

Reply:
Sure! 08c6376